refinery/analysis_manager/utils.py

```diff
@@ -200,9 +69,9 @@ def fetch_objects_required_for_analysis(validated_analysis_config):
     :return: dict w/ mapping to the commonly used objects
     :raises: RuntimeError
     """
-    study_uuid = validated_analysis_config["studyUuid"]
+    study_uuid = validated_analysis_config["study_uuid"]
```

Comment:
Thinking about the dependency between this string and the json schema, it would probably be overkill to have the json created by python code?